pyvista/demos/demos.py

```diff
@@ -45,7 +45,7 @@ def glyphs(grid_sz=3):
 
     # get dataset where to put glyphs
     x, y, z = np.mgrid[:grid_sz, :grid_sz, :grid_sz]
-    mesh = pv.StructuredGrid(x, y, z)
+    mesh = pv.StructuredGrid(x, y, z, force_float=False)
```

Comment:
Is the `force_float=False` here to prevent the warning? We could just convert `grid_sz` to a float and that would return float-dtype arrays from `mgrid`.